starboard/contrib/linux/stadia/gyp_configuration.py

```diff
@@ -35,7 +35,7 @@ def GetTargetToolchain(self, **kwargs):
     return self.GetHostToolchain(**kwargs)
```

Comment:
Can we just remove stadia files at this point?

Reply:
I put up a separate PR for that last night: https://github.com/youtube/cobalt/pull/168

Don't want to mix it here.